notebooks/filechecker/argo-file-checker.ipynb

```diff
@@ -0,0 +1,385 @@
+    "- The Argo NetCDF format is defined in the [Argo User’s Manual](http://dx.doi.org/10.13155/29825).  \n",
+    "- More details and documentation are available on the [Argo Data Management website](https://www.argodatamgt.org/Documentation).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "caa8969d",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d687b649",
+   "metadata": {},
+   "source": [
+    "## The main steps to run the checker\n",
+    "\n",
+    "1. Setup \n",
+    "    - before running this notebook, install dependencies with:\n",
+    "    ```bash\n",
+    "    pip install -r notebooks/requirements.txt\n",
+    "    ```\n",
+    "2. Run the complete notebook\n",
+    "    - Imports necessary packages.\n",
+    "    - Configures the API URL and DAC.\n",
+    "        - Set the API_BASE_URL to where the File Checker API is running.\n",
+    "        - If you are running the File Checker locally inside Docker, set it to the address where the \n",
+    "            Docker container is exposing the API `http://localhost:8000`\n",
+    "        - If you are running the File Checker in a Kubernetes cluster, use the cluster\n",
+    "            URL where the service is exposed. For example:\n",
+    "            `https://livkrakentst.clusters.bodc.me/ewetchy/amrit/argo-toolbox/api/file-checker`\n",
+    "        - Only one of the URLs should be uncommented based on your environment.\n",
+    "    - Sets the default DAC for validation.\n",
+    "    - Checks the connectivity to the API.\n",
+    "5. Checking files\n",
+    "    - <p>If you need to check few selected files , select them by clicking on the <b>Upload</b> button.</p>\n",
+    "6. Optionally run the checker on all files of a deployment.\n",
+    "    - <p>If you want to check a whole deployment, select the path of the files and select <b> Run Deployment Check </b>.</p>\n",
+    "7. Results are saved to a csv file.Result of first few are shown on the console.\n",
+    "    - The name of the output file is part of configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4ddcb01a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ===============================\n",
+    "# 1️⃣ Importing necessary packages\n",
+    "# ===============================\n",
+    "\n",
+    "import logging\n",
+    "import tempfile\n",
+    "import time\n",
+    "from pathlib import Path\n",
+    "from typing import Any\n",
+    "\n",
+    "import ipywidgets as widgets\n",
+    "import pandas as pd\n",
+    "import requests\n",
+    "from IPython.display import display as ipy_display\n",
+    "from ipywidgets import FileUpload, Label, Output, VBox\n",
+    "\n",
+    "logging.basicConfig(level=logging.INFO, format=\"%(levelname)s: %(message)s\")\n",
+    "logger = logging.getLogger(__name__)\n",
+    "for handler in logger.handlers[:]:\n",
+    "    logger.removeHandler(handler)\n",
```

Comment:
What's going on here?

I'm not convinced that using `logging` is even the correct approach in a notebook.

By default, it will always output to STDERR, which will appear like red error text, which coupled with `__name__` (which probably resolves to `"__main__"` in a notebook) is going to be very confusing for an end-user, I think.